src/main/java/wolox/training/model/User.java

```diff
@@ -0,0 +1,110 @@
+package wolox.training.model;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @Column(nullable = false)
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
This isn't the correct CascadeType. Think that a book exists independantly of this collection. So the only thing you want this collection to do, is to maintain the join table updated. Do you remember what CascadeTypes shall be used for this?

Reply:
My bad! This is totally wrong, the remove operation could make a mess for example
I think that the correct way to do this is with the MERGE type (To merge both entities at the same time) and the PERSIST type (So if I create a Book and I added to a user, then I can save the user and the book will be persist before)